revapi-reporter-file-base/src/test/java/org/revapi/reporter/file/AbstractFileReporterTest.java

```diff
@@ -81,15 +82,10 @@ public void testExistingFileAsOutput() throws Exception {
     public void testKeepEmptyFiles() throws Exception {
         Path file = Files.createTempFile(null, null);
         try (Reporter reporter = new Reporter()) {
-            StringBuffer buf = new StringBuffer();
-            buf.append("{");
-            buf.append("\"output\": \"").append(file.toString()).append("\"");
-            buf.append(",");
-            buf.append("\"keepEmptyFile\": \"").append("false").append("\"");
-            buf.append("}");
-
-            reporter.initialize(
-                    AnalysisContext.builder().build().copyWithConfiguration(JSONUtil.parse(buf.toString())));
+            ObjectNode configuration = JsonNodeFactory.instance.objectNode().put("output", file.toString())
```

Comment:
Made these testing changes as I used a Windows machine to develop and something with the file name was causing invalid JSON errors. Using Jackson ObjectNode directly to create the JSON node as it'll properly encode the string if there are invalid JSON string characters.